charts/linstor-cluster/templates/linstorcluster.yaml

```diff
@@ -0,0 +1,21 @@
+{{- with .Values.linstorCluster }}
+apiVersion: piraeus.io/v1
+kind: LinstorCluster
+metadata:
+  name: {{ .name }}
```

Comment:
let's make all the resources to have template `{{ include "linstor-cluster.fullname" . }}` as name or as prefix to their names.